app/views/users/service_providers/show.html.slim

```diff
@@ -8,29 +8,41 @@ h1 = t('headings.service_providers.show', name: service_provider.name)
 
 table.horizontal-headers
   tr
-    th Issuer
-    td.issuer = service_provider.issuer
-  tr
-    th Name
-    td.name = service_provider.name
+    th Friendly Name
+    td.friendly_name = service_provider.friendly_name
   tr
     th Description
     td.description = service_provider.description
   tr
-    th Metadata URL
-    td.metadata_url = service_provider.metadata_url
+    th Issuer
+    td.issuer = service_provider.issuer
+  tr
```

Comment:
I hadn't seen those dev docs before -- helpful to know.

If we are going to rely on the `agency` being a string for grouping service providers together, then perhaps we need to create a model for that on the dashboard side for ensuring consistency in naming, etc. A single typo would break the grouping.

Also, an `Agency` model could function much like a traditional Organization and allow us to group `User` records together for authorization purposes. I.e. UserA and UserB are in the same `Agency` and so may edit each other's records.